lib/components/sprite_component.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:ui' show Canvas, Paint;
+
+import '../sprite.dart';
+import 'component.dart';
+import 'position_component.dart';
+
+/// A [PositionComponent] that renders a single [Sprite] at the designated position,
+/// scaled to have the designated size and rotated to the designated angle.
+///
+/// This is the most commonly used child of [Component].
```

Comment:
Also, flutters docs rearely use "parent/child" terminology for class inheritance, instead, you can use "subclass".